zk_toolbox/crates/zk_supervisor/src/commands/test/integration.rs

```diff
@@ -63,3 +108,37 @@ fn build_test_contracts(shell: &Shell, ecosystem_config: &EcosystemConfig) -> an
     spinner.finish();
     Ok(())
 }
+
+#[derive(Deserialize)]
+struct TestWallets {
+    #[serde(rename = "web3_url")]
+    _web3_url: String,
+    #[serde(rename = "mnemonic")]
+    _mnemonic: String,
+    base_path: String,
+    #[serde(flatten)]
+    wallets: HashMap<String, String>,
+}
+
+async fn fund_test_wallet(
+    ecosystem: &EcosystemConfig,
+    chain: &ChainConfig,
+    address: H160,
+) -> anyhow::Result<()> {
+    let wallets = ecosystem.get_wallets()?;
```

Comment:
I'd rather fund it from the base path of test mnemonic, rather than from operator. operator could do some operations already and you will mess up the nonce inside the server

Reply:
Addressed in https://github.com/matter-labs/zksync-era/pull/2594/commits/93871bfa2b38b70a3f95f39378d5894f96b6e5ff